playbooks/build_image.yaml

```diff
@@ -0,0 +1,105 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: no
+  tasks:
+  - name: Require openshift_gcp_base_image
+    fail:
+      msg: "A base image name or family is required for image building.  Please ensure `openshift_gcp_base_image` is defined."
+    when: openshift_gcp_base_image is undefined
+
+- name: Launch image build instance
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  tasks:
+    - name: Set facts
+      set_fact:
+        openshift_node_bootstrap: True
+
+    - name: Create the image instance disk
+      gce_pd:
+        service_account_email: "{{ (lookup('file', openshift_gcp_iam_service_account_keyfile ) | from_json ).client_email }}"
+        credentials_file: "{{ openshift_gcp_iam_service_account_keyfile }}"
+        project_id: "{{ openshift_gcp_project }}"
+        zone: "{{ openshift_gcp_zone }}"
+        name: "{{ openshift_gcp_prefix }}build-image-instance"
+        disk_type: pd-ssd
+        image: "{{ openshift_gcp_base_image }}"
+        size_gb: 10
+        state: present
+
+    - name: Launch the image build instance
+      gce:
+        service_account_email: "{{ (lookup('file', gce_service_account_keyfile ) | from_json ).client_email }}"
+        credentials_file: "{{ gce_service_account_keyfile }}"
+        project_id: "{{ openshift_gcp_project }}"
+        zone: "{{ openshift_gcp_zone }}"
+        machine_type: n1-standard-1
+        instance_names: "{{ openshift_gcp_prefix }}build-image-instance"
+        state: present
+        tags:
+          - build-image-instance
+        disk_auto_delete: false
+        disks:
+          - "{{ openshift_gcp_prefix }}build-image-instance"
+      register: gce
+
+    - name: add host to nodes
+      add_host:
+        hostname: "{{ item.public_ip }}"
+        groupname: nodes
+      with_items: "{{ gce.instance_data }}"
+
+    - name: Wait for instance to respond to SSH
+      wait_for:
+        delay: 1
+        host: "{{ item.public_ip }}"
+        port: 22
+        state: started
+        timeout: 120
+      with_items: "{{ gce.instance_data }}"
+
+- name: normalize groups
+  include: /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/initialize_groups.yml
+
+- name: run the std_include
+  include: /usr/share/ansible/openshift-ansible/playbooks/common/openshift-cluster/evaluate_groups.yml
+
+- name: run the std_include
```

Comment:
```
initialize the facts
```

Reply:
I copied this from the current state of openshift-ansible, is this fixed in your PR?  Will recopy from there.

Reply:
I'll fix in my PR.  This has been refactored out from under me.  I'll include these fixes in my PR.